step.yml

```diff
@@ -143,10 +143,10 @@ inputs:
           https://docs.atlassian.com/bitbucket-server/rest/5.16.0/bitbucket-rest.html#idm8287391664
       is_required: true
       
-  - BRANCH_FROM: 
+  - SOURCE_BRANCHES: 
```

Comment:
Can you keep old name for backward compatibility?

Reply:
Got it